### PR TITLE
Bring changes from preprocessing-base

### PR DESCRIPTION
**Pull request by jraddaoui**
Trying to align the existing preprocessing projects, any feedback is welcome and will be added to the base and SFA repositories.

Reviewing commit by commit will make more sense.

### CODE REVIEW

File: internal/config/config.go [hunk 1]
```diff
@@ -41,8 +41,8 @@ func Read(config *Configuration, configFile string) (found bool, configFileUsed
 	v.AddConfigPath(".")
 	v.AddConfigPath("$HOME/.config/")
 	v.AddConfigPath("/etc")
-	v.SetConfigName("preprocessing_moma")
-	v.SetEnvPrefix("PREPROCESSING_MOMA")
+	v.SetConfigName("preprocessing")
+	v.SetEnvPrefix("PREPROCESSING")
```
Comment (djjuhasz) [inline internal/config/config.go:45]: Maybe "SDPS_PREPROCESSING" or "ENDURO_PREPROCESSING" would be better to link preprocessing to Enduro and provide more distinct namespacing?

**Review by djjuhasz — APPROVED** [review]
@jraddaoui I had one minor suggestion for the env var namespacing, but it's not a blocker.

File: internal/config/config.go (hunk 1 above)
Comment (jraddaoui) [reply internal/config/config.go:45]: Cool, I agree, ENDURO_PREPROCESSING looks good to me.

**Comment by jraddaoui** [comment]
I'm looking at that integration test. Not as easy as just changing/removing the invalid log level.

**Comment by jraddaoui** [comment]
In the end it was only that log level, I don't know why the sampledata dir/file had different permissions for me locally.

**Pull request merged**